library/common/extensions/filter/c_types.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "library/common/types/c_types.h"
+
+// NOLINT(namespace-envoy)
+
+/**
+ * Return codes for header filter invocations. See envoy/http/filter.h
+ */
+typedef enum {
+  ENVOY_FILTER_HEADERS_STATUS_CONTINUE,
+  ENVOY_FILTER_HEADERS_STATUS_STOP_ITERATION,
+  ENVOY_FILTER_HEADERS_STATUS_CONTINUE_AND_END_STREAM,
+  ENVOY_FILTER_HEADERS_STATUS_STOP_ALL_ITERATION_AND_BUFFER,
+} envoy_filter_headers_status_t;
```

Comment:
we discussed having a subset of these enums. Do you want to do that here in the declarations of in the mapping functions?

Reply:
I decided to expose them all at this layer because there's no real reason not to.

Reply:
Even if we don't want to expose all of that functionality to platform filters?

Reply:
I think it's fine to expose them here. To explain myself better, my impulse is to not make assumptions at this layer.